packages/jedi/test_jedi.py

```diff
@@ -1,11 +1,10 @@
-def test_jedi(selenium_standalone):
-    selenium_standalone.load_package("jedi")
-    result = selenium_standalone.run(
-        """
-        import jedi
-        script = jedi.Script("import json\\njson.lo", path='example.py')
-        completions = script.complete(2, len('json.lo'))
-        [el.name for el in completions]
-        """
-    )
-    assert result == ["load", "loads"]
+from tools.testing import run_in_pyodide
```

Comment:
Tools is not really a place for python modules. I would rather we put it in `pyodide_build`. I mean it's called build, but in practice it's anything that shouldn't be included in WASM VM (build, testing, etc). So can also rename it to be more general.

Among other things mypy and pytest are not currently run on the `tools` folder.